tests/utils/test-data-generator.ts

```diff
@@ -333,6 +358,41 @@ export class TestDataGenerator {
     };
   };
 
+  public static async generateGrantCreate(input?: GenerateGrantCreateInput): Promise<GenerateGrantCreateOutput> {
```

Comment:
Added this as a convenience, might add the rest in a separate PR and clean up usage of it manually. Makes for reading the tests much cleaner.

Reply:
> Makes for reading the tests much cleaner.

No push back for this convenience method 'coz it is probably a matter of preference and it does make the test look cleaner. Though I want to discuss a bit more with you as my usage on `generateXyz` message methods in `TestDataGenerator` has changed over time: I started out using it very heavily during initial phase of DWN development as it allows defaults in many places, but over time I use it less and it's been many months (possibly more than a year) since I last used it to generate DWN messages. The reason is probably because test scenarios are now way more nuanced/specific and defaults just don't cut it most of the time. And I'd prefer to go straight to the source create method than using a wrapper if it just saves me 1 or two defaults (in this case the expiry?), so I can be sure what gets passed in. I found it harder to debug using the generator method when tests fail because I have to reason through the default logic in the generator method, and some of those logic got complicated themselves over time... 🤣 

Again no push back, just wanted to have that discussion to give you context on why I started stay away from the generate methods.

Reply:
Yeah that makes sense! I think my favorite part about the `generateXyz` is the data stream, it gets ugly sometimes with long variable names 🤣 

Would you prefer I remove this and just create the grants using the normal records create where I'm using them?

Reply:
Nah, we good as we on the same page regarding pros and cons.